index.html

```diff
@@ -0,0 +1,155 @@
+            <a href="">Logout</a>
+          </div>
+        </div>
+        <div class="main-content-container flex-col p-2">
+          <div class="main-content-section1-container flex-row mb-1">
+            <div class="user-image-container component-image-rounded">
+              <img
+                class="user-image"
+                src="/images/avatar.svg"
+                alt="user-image"
+              />
+            </div>
+            <div class="user-details-container">
+              <div class="user-details-section1-container">
+                <h2>Priya Patel</h2>
+                <small>priya@gmail.com</small>
+              </div>
+              <div class="user-details-section2-container mt-2 flex-row">
+                <div class="total-score-container flex-row">
+                  <i class="fa-solid fa-flag text-md"></i>
+                  <div class="flex-col align-center">
+                    <h2>27</h2>
+                    <p>Total score</p>
+                  </div>
+                </div>
+                <div class="total-score-container flex-row">
+                  <i class="fa-solid fa-clock text-md"></i>
+                  <div class="">
+                    <h2>27min</h2>
+                    <p>Total time</p>
+                  </div>
+                </div>
+                <div class="total-score-container flex-row">
+                  <i class="fa-solid fa-circle-check text-md"></i>
+                  <div class="flex-col align-center">
+                    <h2 class="">200</h2>
+                    <p>Answers</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="main-content-section2-container p-1 flex-row">
+            <div class="main-content-section2-card-container">
+              <h2 class="py-2 mb-2 flex-row justify-center">Achievements</h2>
+              <div class="user-achievements-container flex-col">
+                <div class="flex-row justify-around p-2">
+                  <div class="trophy-container component-image-square">
+                    <img src="/images/medal.png" alt="medal-image" />
+                    <p>1st Prize</p>
+                  </div>
+                  <div class="trophy-container component-image-square">
+                    <img src="/images/medal (1).png" alt="medal-image" />
+                    <p>2nd Prize</p>
+                  </div>
+                </div>
+                <div class="space-1rem"></div>
+                <div class="trophy-container component-image-square">
+                  <img src="/images/medal (3).png" alt="medal-image" />
+                  <p>3rd Prize</p>
+                </div>
+                <div class="space-1rem"></div>
+              </div>
+            </div>
+            <div class="main-content-section2-card-container">
+              <h2 class="py-2 flex-row justify-center">Featured Category</h2>
```

Comment:
py-2?

Reply:
it stands for padding along y-axis if still doubt persists go through tailwind CSS .